### PR TITLE
Add quay #160

### PR DESCRIPTION
**Pull request by DanielFroehlich**
Adding basic quay deployment.

### CODE REVIEW

**Comment by DanielFroehlich** [comment]
I think I fucked up on creating the branch, there are two commits from my previous PR regarding mtv in this one here, too. probably better to accept that first PR first, and then I rebase this one here. Sorry for the nose.

File: isar-apps/mtv/05_externalsecret-rhv.yaml
Comment (rbo) [inline isar-apps/mtv/05_externalsecret-rhv.yaml:1]: Looks like you mixed the MTV & Quay stuff a bit.

File: isar-apps/quay/read.me
Comment (rbo) [inline isar-apps/quay/read.me:1]: Content doesn't match the folder

**Review by rbo — CHANGES_REQUESTED** [review]
What's the storage backend? Default storage-class? Or odf?

File: isar-apps/mtv/05_externalsecret-rhv.yaml
Comment (DanielFroehlich) [reply isar-apps/mtv/05_externalsecret-rhv.yaml:1]: yup, see my earlier comment. let me try to rebase now that the mtv part is in.

**Comment by DanielFroehlich** [comment]
> What's the storage backend? Default storage-class? Or odf?

Default ObjectStorage class.  i.e. noobaa and in the current config. 
Happy to change that to something else, just let me know which S3 you want to use.

File: isar-apps/quay/read.me
Comment (DanielFroehlich) [reply isar-apps/quay/read.me:1]: good catch, did forget to save. better now?

**Comment by rbo** [comment]
> > What's the storage backend? Default storage-class? Or odf?
> 
> Default ObjectStorage class. i.e. noobaa and in the current config. Happy to change that to something else, just let me know which S3 you want to use.

Oh, you are right, I forgot that. 

I don't have an opinion on it yet. I don't know how stable ODF is on the that cluster. Since it is deployed, it's quite stable. Comparing to MetalLB or external secret operator :-D

**Pull request merged**